modules/client/src/client-common/modules/ipfs.ts

```diff
@@ -3,48 +3,61 @@ import { Context } from "../../client-common/context";
 import { IClientIpfsCore } from "../interfaces/core";
 import { Client as IpfsClient } from "@aragon/sdk-ipfs";
 
-export class IPFSModule implements IClientIpfsCore {
-  private _ipfs: IpfsClient[] = [];
-  private _ipfsIdx: number = -1;
+const IPFSMap = new Map<IPFSModule, IpfsClient[]>();
+const IPFSIdxMap = new Map<IPFSModule, number>();
```

Comment:
Same, lowercase. And maybe a more maningful name like `ipfsClientsMap` and `ipfsClientIdxMap`